pkg/cmd/status.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+
+	cmdutil "github.com/cloud-native-application/rudrx/pkg/cmd/util"
+
+	"github.com/ghodss/yaml"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func NewAppStatusCommand(f cmdutil.Factory, c client.Client, ioStreams cmdutil.IOStreams) *cobra.Command {
```

Comment:
I think you could try to write a unit test for this function, then you will understand why I suggest use `RunE`

Reply:
Okay, let me fix workload_run_test.go and trait_bind_test.go first.